behave/contrib/csv_table_from_file.py

```diff
@@ -1,43 +1,83 @@
-import logging
+# -*- coding: UTF -*-
+# pylint: disable=line-too-long
+"""
+Provides functionality to dynamically populate Examples tables in Behave
+feature files from CSV files based on @from_file tags.
+
+EXAMPLE:
+
+.. sourcecode:: gherkin
+
+    # -- FILE: features/example.feature
+
+    Feature: Example Feature
+
+    Scenario Outline: Example Scenario
+        Given get <username> and <email> and <password>
+
+        @from_file=testdata.csv
+        Examples:
+          | username | email   | password |
+          | .        | .       | .        |
```

Comment:
The second table row without any values (or using dots) is not really necessary (I think).

Reply:
Resolved